app/advancedfilters/evidenceFilter.html

```diff
@@ -7,8 +7,7 @@
         <li ng-repeat="eco in ecos" class="checkbox">
           <label>
             <input type="checkbox" ng-model="eco.checked">
-            <a ng-click="showEvidenceCodeOntologyGraph('{{eco.id}}')">{{eco.id}}</a>
-            <strong>{{eco.name}}</strong> {{eco.description}}
+            <term-line id="eco.id" name="eco.name"></term-line> {{eco.description}}
```

Comment:
Because the GO terms don't have a description - the term-line is only id/name + action buttons